.circleci/config.yml

```diff
@@ -182,43 +87,56 @@ jobs:
       - run: carthage build --no-skip-current && for platform in Mac iOS tvOS watchOS; do test -d Carthage/Build/${platform}/BonMot.framework || exit 1; done
 
   deploy-to-cocoapods:
+    executor: xcode-10
+    steps:
+      - setup
+      - run:
+          name: Install CocoaPods Specs Repo
+          command: |
+            curl https://cocoapods-specs.circleci.com/fetch-cocoapods-repo-from-s3.sh | bash -s cf
+      - run: bundle exec pod trunk push
+
+
+executors:
+  xcode-10:
     macos:
       xcode: "10.2.1"
     environment:
       LC_ALL: en_US.UTF-8
       LANG: en_US.UTF-8
-      BUNDLE_JOBS: 3
-      BUNDLE_RETRY: 3
-      BUNDLE_PATH: /Users/distiller/project/build/gems
       HOMEBREW_NO_AUTO_UPDATE: 1
-      SWIFT_VERSION: 5
     shell: /bin/bash --login -eo pipefail
-    working_directory: /Users/distiller/project
+
+commands:
+  setup:
+    description: "Shared setup"
+    steps:
+      - checkout
+      - restore-gems
+
+  restore-gems:
+    description: "Restore Ruby Gems"
     steps:
       - run:
           name: Set Ruby Version
-          command:  echo "ruby-2.4" > ~/.ruby-version
-      - checkout
-      # Setup Ruby dependencies
+          command:  echo "ruby-2.5" > ~/.ruby-version
       - restore_cache:
-          keys:
-            - 2-gems-{{ checksum "Gemfile.lock" }}
-            # Fall back to a previous cache if exact match cannot be found
-            - 2-gems-
-      - run:
-          name: Restore Ruby Gems
-          command: |
-            bundle check || bundle install
-      # Cache Gems
+          key: 1-gems-{{ checksum "Gemfile.lock" }}
+      - run: bundle check || bundle install --path vendor/bundle
       - save_cache:
-          key: 2-gems-{{ checksum "Gemfile.lock" }}
+          key: 1-gems-{{ checksum "Gemfile.lock" }}
           paths:
-            - "/Users/distiller/project/build/gems"
-      - run:
-          name: Install CocoaPods Specs Repo
-          command: |
-            curl https://cocoapods-specs.circleci.com/fetch-cocoapods-repo-from-s3.sh | bash -s cf
-      - run: bundle exec pod trunk push
+            - vendor/bundle
+
+  lint-pod:
+    description: "Lints podspec with specified Swift version"
+    parameters:
+      swift-version:
+        type: string
+        default: "5.0"
+    steps:
+      - run: bundle exec pod repo add-cdn jsDelivr-Specs "https://cdn.jsdelivr.net/cocoa/"
```

Comment:
#? why this repo, in particular?

Reply:
It's the only Specs CDN so far: http://blog.cocoapods.org/CocoaPods-1.7.0-beta/

Reply:
Ohh, got it. I thought it was going to be a system where any company could sign up to host a mirror, but it’s actually just one free CDN, which is sponsored by a bunch of companies. Smart. https://www.jsdelivr.com

Reply:
My guess is that once it's no longer experimental there might be some other CDN mirrors pop up. CDN Specs still doesn't support some use cases like `trunk push`, as far as I'm aware.